cardano-cli/test/cardano-cli-golden/files/golden/help/conway_governance_action_create-update-proposal.cli

```diff
@@ -0,0 +1,130 @@
+                                                                     [--min-fee-linear LOVELACE]
+                                                                     [--min-utxo-value NATURAL]
+                                                                     [--key-reg-deposit-amt NATURAL]
+                                                                     [--pool-reg-deposit NATURAL]
+                                                                     [--min-pool-cost NATURAL]
+                                                                     [--pool-retirement-epoch-boundary EPOCH_BOUNDARY]
+                                                                     [--number-of-pools NATURAL]
+                                                                     [--pool-influence RATIONAL]
+                                                                     [--monetary-expansion RATIONAL]
+                                                                     [--treasury-expansion RATIONAL]
+                                                                     [--utxo-cost-per-word LOVELACE]
+                                                                     [--price-execution-steps RATIONAL
+                                                                       --price-execution-memory RATIONAL]
+                                                                     [--max-tx-execution-units (INT, INT)]
+                                                                     [--max-block-execution-units (INT, INT)]
+                                                                     [--max-value-size INT]
+                                                                     [--collateral-percent INT]
+                                                                     [--max-collateral-inputs INT]
+                                                                     [--utxo-cost-per-byte LOVELACE]
+                                                                     --governance-action-deposit NATURAL
+                                                                     ( --stake-pool-verification-key STRING
+                                                                     | --cold-verification-key-file FILE
+                                                                     | --stake-pool-id STAKE_POOL_ID
+                                                                     | --stake-verification-key STRING
+                                                                     | --stake-verification-key-file FILE
+                                                                     | --stake-key-hash HASH
+                                                                     )
+                                                                     [--cost-model FILE]
+                                                                     --out-file FILE
+
```

Comment:
Does not seem to include protocol parameters introduced with Conway. 

- governance voting thresholds ($P1$, $P{2a}$, $P{2b}$, $P3$, $P4$, $P{5a}$, $P{5b}$, $P{5c}$, $P6$, $P7$, $Q1$, $Q{2b}$, $Q_4$)
- constitutional committee term limits
- governance action expiration
- governance action deposit (govDeposit)
- DRep deposit amount (drepDeposit)
- DRep activity period (drepActivity)
- minimal constitutional committee size
- maximum term limit (in epochs) of the constitutional committee